docker/release/base/Dockerfile.in

```diff
@@ -2,6 +2,13 @@ FROM fedora:34 AS keylime_base
 LABEL version="_version_" description="Keylime Base - Only used as an base image for derived packages"
 MAINTAINER Keylime Team <main@keylime.groups.io>
 
+RUN dnf -y install dnf-plugins-core git && dnf -y builddep tpm2-tools
+RUN git clone https://github.com/tpm2-software/tpm2-tools.git && \
```

Comment:
that means that I need to update mine too ... : )))

BTW, would make more sense to checkout tag 5.4? Using master / main will generate problems

Reply:
Yes I'll change that.